tests/regression/gmail/test_firefox_gmail_ail_type_in_reply_field.sikuli/test_firefox_gmail_ail_type_in_reply_field.py

```diff
@@ -39,13 +39,13 @@
 app.wait_for_loaded()
 
 # Wait for stable
-sleep(5)
+sleep(2)
 
 # PRE ACTIONS
 app.click_first_mail()
```

Comment:
I suggest that you can wait for reply button to show up for `app.click_first_mail()` and wait for submit button to show up for `app.click_reply_btn()`. Sometimes, the could still not fully loaded after a short wait, and the system will do the ` com.system_print('[log]  TYPE "a"')` right away. It could cause error and I did met it in Facebook cases.